app/helpers/admin/site_maps_helper.rb

```diff
@@ -7,7 +7,7 @@ module Admin::SiteMapsHelper
   #
   # @!visibility private
   def sitemap_pages
-    @sitemap_pages ||= proper_pages.only(:id, :name, :page_id, :path).group_by(&:page_id)
+    @sitemap_pages ||= proper_pages.only(:id, :name, :page_id, :path, :position, :role).group_by(&:page_id)
```

Comment:
Ah ok, yeah I forgot this was changed recently. So does that mean `master` is currently broken? In which case should we merge this fix first, before rails 4, and do a cheeky point release? Otherwise all the sites we're going to leave on rails 3 will pick this up and be broken.